src/estimation/src/BerdyHelper.cpp

```diff
@@ -1366,6 +1401,35 @@ bool BerdyHelper::computeBerdySensorMatrices(SparseMatrix<iDynTree::ColumnMajor>
         // bY for the joint wrenches is zero
     }
 
+    ////////////////////////////////////////////////////////////////////////
+    ///// COM ACCELERATION SENSOR
+    ////////////////////////////////////////////////////////////////////////
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+        // Get the row index corresponding to the com accelerometer sensor
+        IndexRange comAccelerometerRange = this->getRangeCoMAccelerometerSensorVariable(COM_ACCELEROMETER_SENSOR);
+
+        for(LinkIndex idx = 0; idx < static_cast<LinkIndex>(m_options.comConstraintLinkIndexVector.size()); idx++)
+        {
+            // Get the column index corresponding to the net link external wrench sensor
+            IndexRange netExternalWrenchSensor = this->getRangeLinkSensorVariable(NET_EXT_WRENCH_SENSOR,
+                                                                                  static_cast<LinkIndex>(m_options.comConstraintLinkIndexVector.at(idx)));
+
+            iDynTree::Rotation base_R_link = base_H_links(idx).getRotation();
```

Comment:
following the comment above, here we should use `linkIdx` instead of `idx`

Reply:
Updated in https://github.com/robotology/idyntree/pull/571/commits/8b9ffe5c2fc2d8b718da6a8a4fec0a2780eb6c5e